### DISCUSSION TITLE
Docs: Show UI of fields in the docs

### DISCUSSION THREAD

**Discussion by NeeshSamsi** [Thinkmill/keystatic#316] [Ideas]
I was going through the docs to explore Keystatic in a side project. The fields are all documented well, but I am not familiar with some of the types like Blocks, Path references, Conditionals, etc. The docs already show the code implementation and type signature. It would be hugely beneficial if the docs included a couple of screenshots or a short gif of the UI that the field generates, to understand its used cases and just see what exactly it looks like on the UI side.

**Comment by simonswiss** [comment]
Hey!

That's a really nice suggestion, and I agree with you. This is something we've discussed internally, and will definitely try to improve.

We were also contemplating making Stackblitz examples for each field, so you can see the UI and play with the schema instead of just looking at a screenshot.

Stay tuned, and thanks for the feedback/suggestion 🙏

**Reply by NeeshSamsi** [reply]
Awesome! Can't wait

**Reply by hauptrolle** [reply]
Some fields now have examples. See: https://keystatic.com/docs/fields/multiselect
